File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/PermissionResVo.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

import java.io.Serializable;
import java.util.List;

@Data
public class PermissionResVo implements Serializable {
    private Integer loginRole;
    private List<CheckboxesVo> checkboxes;

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/PermissionResVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import java.io.Serializable


@AllOpen
class PermissionResVo : Serializable {
    var loginRole: Int? = null
    var checkboxes: List<CheckboxesVo>? = null
}
